File: problems/1953/solution.py
import solution


class Solution(solution.Solution):
    def solve(self, test_input=None):
        return self.numberOfWeeks(list(test_input))

    def numberOfWeeks(self, milestones):
        """
        :type milestones: List[int]
        :rtype: int
        """
        m, s = max(milestones), sum(milestones)
        return (s - m) * 2 + 1 if m > s - m else s

File: problems/1953/solution.go
package problem1953

import (
	"encoding/json"
	"log"
	"strings"
)

func numberOfWeeks(milestones []int) int64 {

}

func Solve(input string) interface{} {
	values := strings.Split(input, "\n")
	var milestones []int

	if err := json.Unmarshal([]byte(values[0]), &milestones); err != nil {
		log.Fatal(err)
	}

	return numberOfWeeks(milestones)
}
